cardano-cli/src/Cardano/CLI/Types/Common.hs

```diff
@@ -665,3 +667,12 @@ data PotentiallyCheckedAnchor anchorType anchor
   -- ^ Whether to check the hash or not (CheckHash for checking or TrustHash for not checking)
   }
   deriving (Eq, Show)
+
+_pcaAnchor
```

Comment:
Please remove this. We don't need lenses.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/937/commits/5666bd6559dd85669e7ada431bdb8730f534d857